lib/angulartics-newrelic-insights.js

```diff
@@ -16,6 +16,7 @@
     .config(['$analyticsProvider', function($analyticsProvider) {
       angulartics.waitForVendorApi('newrelic', 100, function(newrelic) {
         $analyticsProvider.registerEventTrack(function(action, properties) {
+          properties.eventType = 'PageAction';
```

Comment:
On further reflection, mutating an object that we don't own seems like a bad idea.